src/test/rpc/NoRippleCheck_test.cpp

```diff
@@ -254,27 +254,36 @@ class NoRippleCheckLimits_test : public beast::unit_test::suite
         env (fset (alice, asfDefaultRipple));
         env.close ();
 
-        for (auto i = 0; i < ripple::RPC::Tuning::noRippleCheck.rmax + 5; ++i)
+        auto checkBalance = [&env]()
         {
-            if (! admin)
+            // this is endpoint drop prevention. Non admin ports will drop
+            // requests if they are coming too fast, so we manipulate the
+            // resource manager here to reset the enpoint balance (for
+            // localhost) if we get too close to the drop limit. It would
+            // be better if we could add this functionality to Env somehow
+            // or otherwise disable endpoint charging for certain test
+            // cases.
+            using namespace ripple::Resource;
+            using namespace std::chrono;
```

Comment:
If you want to narrow the scope of this `using namespace std::chrono`, I think you can move it inside the braces of the
```
if (c.balance() > warningThreshold)
```
Just a thought.